cli/src/main/java/com/devonfw/tools/ide/tool/intellij/Intellij.java

```diff
@@ -68,6 +63,56 @@ protected void postInstall() {
     EnvironmentVariables envVars = this.context.getVariables().getByType(EnvironmentVariablesType.CONF);
     envVars.set("IDEA_PROPERTIES", this.context.getWorkspacePath().resolve("idea.properties").toString(), true);
     envVars.save();
+    createStartupCommandScript();
+  }
+
+  private void createStartupCommandScript() {
+    this.context.getFileAccess().mkdirs(getToolPath().resolve("bin"));
+    Path bashFile;
+    String bashFileContentStart = "#!/usr/bin/env bash\n'";
+    String bashFileContentEnd = "' $*";
+    if (this.context.getSystemInfo().isMac()) {
+      try {
+        bashFile = Files.createFile(getToolBinPath().resolve(getName()));
+        Files.writeString(bashFile,
+            bashFileContentStart + getToolPath().resolve("IntelliJ IDEA CE.app").resolve("Contents").resolve("MacOS").resolve(IDEA) + bashFileContentEnd);
```

Comment:
Please check how this was done in `devonfw-ide`.
It should go to `IdeToolCommandlet` and the script should simply use IDEasy itself to launch the IDE.
Any other approach is cursed to fail from the start.